sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -765,6 +765,166 @@ public virtual RecognizeInvoicesOperation StartRecognizeInvoicesFromUri(Uri invo
         }
         #endregion
 
+        #region ID Documents
+
+        /// <summary>
+        /// Recognizes values from one or more ID documents.
```

Comment:
Consider changing for 
Analyze ID documents using optical character recognition (OCR) and a prebuilt model trained on ID documents
to extract key information from passports and US driver licenses.